business/health.go

```diff
@@ -77,7 +114,7 @@ func (in *HealthService) getAppHealth(namespace, app, rateInterval string, detai
 	health.Requests = in.getAppRequestsHealth(namespace, app, rateInterval)
 
 	// Deployment status
-	health.DeploymentStatuses = castDeploymentsStatuses(details.Deployments)
+	health.WorkloadStatuses = castWorkloadStatuses(ws)
```

Comment:
Not very important, about the naming it's the same remark I did on the related frontend PR: this "status" just tells how many pods are running versus how many are expected. Maybe we can find something more accurate than "WorkloadStatus". Like "RunningPodsStatus" or "RunningReplicasStatus" or just "ReplicasStatus" or "RunningRatio" ? Lot of possibilities :-) What do you think?

Reply:
Naming is hard. Yeah, if you look the PR, main strategy is to move from "Deployment" to "Workload" in a more generic way.
Perhaps in following iterations it can be refine and change this to ReplicaStatus or other naming as well.